classes/class-editor.php

```diff
@@ -92,30 +92,32 @@ public function load( $settings ) {
 		// Restrict tinymce buttons
 		add_filter( 'tiny_mce_before_init', [ $this, 'tiny_mce_before_init' ] );
 
-		// Gutenberg scripts
-		wp_enqueue_script( 'wp-block-library' );
-		wp_enqueue_script( 'wp-format-library' );
-		wp_enqueue_script( 'wp-editor' );
-		wp_enqueue_script( 'wp-plugins' );
+		if ( ! defined( '__EXPERIMENTAL_DYNAMIC_LOAD' ) ) {
+			// Gutenberg scripts
+			wp_enqueue_script( 'wp-block-library' );
+			wp_enqueue_script( 'wp-format-library' );
+			wp_enqueue_script( 'wp-editor' );
+			wp_enqueue_script( 'wp-plugins' );
```

Comment:
We don't need these four lines anymore. This [line](https://github.com/Automattic/blocks-everywhere/pull/185/files#diff-6b1de222bf946241597a957c0b9abefa1d02f600845d63fcbdc18d605960f481R415) takes care of enqueuing these.